src/server/components/formRunner/proxyMiddleware.ts

```diff
@@ -23,24 +22,12 @@ export function configureFormRunnerProxyMiddleware(server: Express): void {
 
         let data: string = proxyResData.toString("utf8");
 
-        // TODO: Find a better way to do this.
-        if (userReq.baseUrl.includes("/status")) {
```

Comment:
So the block was previously added because we wanted customised status pages at the end of the form flow as the runner didn't support it at the time. Now that's been added I've removed the block and allowed the runner to do it's own thing.

I've not changed anything on the feedback form because I've reverted that block of code to be how it was before I changed anything. We can look to change it separately as we are wanting to get the form runner to run separately instead of the current proxying of it